ui/src/static/js/index.js

```diff
@@ -21,17 +28,61 @@ socket.on("refresh", function (data) {
     }, 5)
 
 });
-socket.on("log", function (data) {
+socket.on("ok", function (data) {
+    let  command = data['command'];
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/PotatoHD404/Laba2/pullRequest?prid=7321542)
```suggestion
    let  command = data["command"];
```